submissions/TaniaBondarenko/dom_task/my_javascript.js

```diff
@@ -0,0 +1,40 @@
+let i = 0; let n = 12;
```

Comment:
It's bette to use `imgs.length` here instead of `12`, to my mind. And I think you don't need `let i = 0;` variable

Reply:
fixed